server/utils/utils.ts

```diff
@@ -14,4 +16,11 @@ const properCaseName = (name: string): string => (isBlank(name) ? '' : name.spli
 const convertToTitleCase = (sentence: string): string =>
   isBlank(sentence) ? '' : sentence.split(' ').map(properCaseName).join(' ')
 
+export const withDefaultTime = (value?: string): string | undefined => {
+  if (value) {
+    return moment(value).format('YYYY-MM-DDTHH:mm:ss')
```

Comment:
I think we're trying to move away from moment, but this is an internal service anyway so fine.